yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -542,44 +566,66 @@ modules by recommended-min.  See <xref target="import_semver" format="default"/>
 </section>
 </section>
 <section anchor="import_semver" numbered="true" toc="default">
-<name>Import Module by Semantic Version</name>
+<name>Import Module by YANG Semantic Version</name>
 <t>
 <xref target="I-D.ietf-netmod-yang-module-versioning" format="default"/>
  allows for imports to be
-    done based on a module or a derived revision of a module.  The rev:recommended-min
-    statement can specify either a revision date or a revision label.  The YANG Semver revision-label value can be used as the argument to rev:recommended-min
-    .  When used as such, any module that contains exactly the same YANG semantic version in its revision history may be used
-    to satisfy the import requirement.  For example:</t>
+    done based on the earliest supported date and later using the rev:recommended-min-date extension.  This section
+    defines a similar extension for controlling import by YANG semantic version, as well as the rules for how
+    imports are resolved.</t>
+<section anchor="recommend_min_version" numbered="true" toc="default">
+<name>The recommended-min-version Extension</name>
+<t>The ietf-yang-semver module defines a "recommended-min-version" extension -- a substatement to the "import" statement -- 
+that takes a YANG semantic version as its argument and specifies
+that the minimum version of the associated module being imported SHOULD be greater than or equal to the specified
+value.  The specific conditions for determining if a module's version is greater than or equal is defined in
+<xref target="semver_import_rules" format="default"/> below.  Multiple recommended-min-version statements 
+MAY be specified.  If there are multiple
+recommended-min-version statements, they are treated as a logical OR.  Removing recommended-min-version
+statements is considered a backwards compatible change.  An example use is:</t>
```

Comment:
In the YANG module we say "Adding, removing or updating a 'recommended-min-version'
       statement to an import is a backwards-compatible change.".  So this should match here, or maybe just remove that detail from here and leave it to the yang module below?